mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/plugins/DependencyInjection.kt

```diff
@@ -86,7 +86,7 @@ private fun db(databaseConfig: DatabaseConfig) = module(createdAtStart = true) {
         FlywayDatabaseAdapter(
             databaseConfig,
             FlywayDatabaseAdapter.MigrationConfig(
-                strategy = FlywayDatabaseAdapter.InitializationStrategy.MigrateAsync
+                strategy = FlywayDatabaseAdapter.InitializationStrategy.Migrate
```

Comment:
Er det en grunn til at det endres til synkron migrering og ikke asynkron?

Reply:
Med asynkron så feiler testene, og så ble den etter det jeg har forstått endret til asynkron når den nye indeksen skulle lages for å slippe å vente på den, så jeg har inntrykket av at den vanligvis skal være synkron? Men er ikke heeelt sikker på om det kanskje burde stilles inn for testene bare 🤔

Reply:
Yes, vi har hatt den som Migrate til vanlig, men satt den til MigrateAsync når vi har deployet databasemigrasjoner som tar lang tid. Default er Migrate, så egentlig kan hele linja endres til `FlywayDatabaseAdapter(databaseConfig)`.

Tidligere har vi ikke hatt tester som faktisk har kjørt opp ktor-applikasjonen så det går greit, men hvis vi får behovet for å deploye en ny async migrasjon så får vi fort trøbbel med de nye testene igjen. Løsningen blir i tilfellet å flytte migrasjonsconfig'en enda lengre ut slik at dette spesifiseres i `application-{local,dev,prod}.yaml` i stedet for at det hardkodes i oppsettet her.

Reply:
Skjønner!